apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -0,0 +1,267 @@
+import { html } from '@prairielearn/html';
+import { run } from '@prairielearn/run';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { Modal } from '../../../components/Modal.html.js';
+import { Navbar } from '../../../components/Navbar.html.js';
+import { QuestionContainer } from '../../../components/QuestionContainer.html.js';
+import { compiledScriptTag, nodeModulesAssetPath } from '../../../lib/assets.js';
+import { type Question, type AiGenerationPrompt } from '../../../lib/db-types.js';
+
+export function AiGenerateEditorPage({
```

Comment:
This should match the name of the page as listed in the filename. We don't typically include `Page` in these function names.

Reply:
Resolved in e9257e2.